extensions/amp-app-banner/1.0/test/test-amp-app-banner.js

```diff
@@ -0,0 +1,98 @@
+import {htmlFor} from '#core/dom/static-template';
+
+import {toggleExperiment} from '#experiments';
+
+import {platformService} from '#preact/services/platform';
+
+import {waitFor} from '#testing/helpers/service';
+import '../amp-app-banner';
+
+describes.realWin(
+  'amp-app-banner-v1.0',
+  {
+    amp: {
+      extensions: ['amp-app-banner:1.0'],
+    },
+  },
+  (env) => {
+    let win;
+    let doc;
+    let html;
+
+    beforeEach(async () => {
```

Comment:
Nit: Splitting these into multiple `beforeEach` is a net negative for readability IMO